qdrant-landing/content/articles/scalar-quantization.md

```diff
@@ -0,0 +1,269 @@
+---
+title: Quantized Qdrant Queries Quality
+short_description: "TBD"
+description: "TBD"
+social_preview_image: /articles_data/scalar-quantization/social_preview.png
+small_preview_image: /articles_data/scalar-quantization/quantization-icon.svg
+preview_dir: /articles_data/scalar-quantization/preview
+weight: 3
+author: XXX
+author_link: XXX
+date: 2023-03-21T10:45:00+01:00
+draft: false
+keywords:
+  - vector search
+  - scalar quantization
+  - TBD
+---
+
+High-dimensional vector embeddings can be memory-intensive, especially when working with 
+large datasets consisting of millions of vectors. Memory footprint really starts being 
+a concern when we scale things up. A small choice of the data type used to store a single
+number impacts even billions of numbers and can drive the memory requirements crazy. The
+higher the precision of your type, the more accurately you can represent the numbers. 
+The more accurate your vectors, the more precise is the distance calculation. But the 
+advantages stop paying off when you need to order more and more memory. 
+
+Qdrant chose `float32` as a default type used to store the numbers of your embeddings. 
+So a single number needs 4 bytes of the memory and a 512-dimensional vector occupies 
+2 kB. That's only the memory used to store the vector. There is also an overhead of the
+HNSW graph, so as a rule of thumb we estimate the memory size with the following formula:
+
+```
+memory_size = number_of_vectors * vector_dimension * 4 bytes * 1.5
+```
+
+While Qdrant offers various options to store some parts of the data on disk, starting
+from the version 1.1.0, you can also optimize your memory by changing the data type used
+by the embeddings. We've implemented the mechanism of **Scalar Quantization**!
```

Comment:
We are not exactly changing the data type of vectors. We still store original. Maybe we could say that we are compressing vectors?
Also, we improve speed as well, not only memory